libs/vscode-ui/components/src/lib/field-items/field-items.pipe.ts

```diff
@@ -0,0 +1,19 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { ItemsWithEnum, Option } from '@nx-console/schema';
+
+export const getOptionItems = (field: Option): string[] | undefined => {
+  return (
```

Comment:
this typing can be a little annoying on items, especially to access these items in the template, so the pipe

Reply:
Nice!